README.md

```diff
@@ -177,11 +179,13 @@ Downgrade `pip` and `setuptools` in virtualenv:
     pip install setuptools==24.0.3
 
 Install node environment in same virtualenv:
-
+    
+    pip install nodeenv==1.1.2
```

Comment:
You're right that at this point `nodeenv` needs to be installed. But actually, `nodeenv==1.1.1` is installed in the following step with `pip install -r requirements/edx/base.txt`. So it would make more sense to move the `nodeenv -p` command right after all `pip install -r` commands.
What do you think?